src/pages/DirectMessage/components/DMChattingList/DMChattingList.Styles.ts

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components"
+
+export const DMChattingListLayout = styled.div`
+  width: 60%;
+  height: 100%;
+  background-color: ${({ theme }) => theme.colors.point};
```

Comment:
point대신 gray40이 더 이쁜 것 같은데 다른 팀원분들 의견도 취합해서 정하면 될 것 같습니다!! 🤔
```suggestion
  background-color: ${({ theme }) => theme.colors.gray40};
```